src/commands/matchmaking.ts

```diff
@@ -0,0 +1,25 @@
+import { Command } from "../lib/Command"
+import config from "../lib/config";
+import { CommandArg, CommandExecuteParameters } from "../types"
+//import bot from "../app" //Need bot/discord client to find the role.
+const options: CommandArg[] = []
+
+const cooldown = 60;
+
+const matchmaking = async (options: CommandExecuteParameters) => {
+  const { type, message, interaction } = options
+  const isSlash = type === "interaction"
+  var response = `<@&${config.matchmaking_role_id}>!`;
+  return isSlash ? interaction?.reply((response + ` Pinged by ${interaction.user.displayName}`)) : message?.reply((response + ` Pinged by ${message.author.displayName}`));
+  //.displayName: https://old.discordjs.dev/#/docs/discord.js/main/class/User?scrollTo=displayName
+  // Apparently doesn't exist acording to ts :shrug:
```

Comment:
.displayName exists on msg.author and interaction.user
it returns "The global name of this user, or their username if they don't have one"